dojo/templates/notifications/engagement_added.tpl

```diff
@@ -3,12 +3,12 @@
 {% if type == 'mail' %}
     Hello,
 
-    the engagement "{{ engagement.product }}" has been created. It can be viewed here: {{ url }}
+    The engagement "{{ engagement.name }}" has been created. It can be viewed here: {{ url }}
```

Comment:
maybe we could still mention the product in parentheses along with the name? wdyt?

Reply:
Agreed. Maybe something like "Engagement {{ engagement.name }} has been created in product {{ enagament.product }}". Kinda choppy, but something of that nature.

Reply:
Makes sense. Changes have been made.